packages/formatting/src/generators/devcontainer.ts

```
@@ -0,0 +1,110 @@
+ * DevContainer configuration generator
+ */
+
+import type { FormatterDetection, DevContainerConfig } from '../types/index.js';
+import { getImageForFormatters } from '../utils/devcontainer.js';
+
+/**
+ * Generate devcontainer.json configuration
+ */
+export function generateDevContainerConfig(
+  formatters: FormatterDetection,
+  preset: string = 'standard',
```

ESLint

[error] 'preset' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)